modules/stroeerCoreBidAdapter.js

```diff
@@ -155,10 +153,14 @@ function groupBy(array, keyFns) {
 
 function getVersionValues (win) {
   const pbjs = getGlobal();
-  return {
+  const versionObject = {
     yl: win.YLHH?.bidder?.settings?.version,
     pb: pbjs.version
   };
+
+  versionObject.mt = win?.SDG?.version && win.SDG.version()
```

Comment:
Oh.. i missed this initially. You can revert to returning as inline object. You can still do this:
```
return {
  yl: win.YLHH?.bidder?.settings?.version,
  pb: pbjs.version,
  mt: win?.SDG?.version && win.SDG.version()
}
```